packages/fluentui/react-northstar/src/components/Button/Button.tsx

```diff
@@ -184,28 +186,37 @@ const Button = compose<'button', ButtonProps, ButtonStylesProps, {}, {}>(
       unstable_props: props,
     });
 
+    const getSlotProps = useSlotProps<ButtonProps>(composeOptions.slotPropsChain, props);
+
     const unhandledProps = useUnhandledProps(composeOptions.handledProps, props);
     const ElementType = getElementType(props);
 
     const renderIcon = () => {
-      return Box.create(icon, {
+      return createShorthand(composeOptions.slots.icon, icon, {
         defaultProps: () =>
           getA11yProps('icon', {
             styles: resolvedStyles.icon,
+            ...getSlotProps('icon'),
           }),
       });
     };
 
     const renderLoader = () => {
-      return Loader.create(loader || {}, {
+      return createShorthand(composeOptions.slots.loader, loader || {}, {
         defaultProps: () =>
           getA11yProps('loader', {
-            role: undefined,
             styles: resolvedStyles.loader,
+            ...getSlotProps('loader'),
           }),
       });
     };
 
+    const renderContent = () => {
+      return createShorthand(composeOptions.slots.content, content, {
+        defaultProps: () => getA11yProps('content', getSlotProps('content')),
```

Comment:
This is extremely confusing 🥺

Reply:
Yeah.. To be honest I never linked that we have to pass all props to the `getA11yProps` function :(

Reply:
Looks better now :)